convert2rhel/toolopts.py

```diff
@@ -202,7 +202,11 @@ def _register_options(self):
             help="A configuration file to safely provide either a user password or an activation key for registering"
             " the system through subscription-manager. Alternatively, passing these values through the"
```

Comment:
The sentence "Alternatively..." is somewhat confusing. Recommend rewriting as:
"This is more secure than passing these values through the --activationkey or --password option, which might leak the values through a list of running processes."